src/controller/order.controller.ts

```diff
@@ -6,12 +6,13 @@ import { Status } from '../enum/status.enum';
 import { getOrderSummary } from '../helpers/order.helper';
 import { IAuthenticatedUser } from '../interfaces/IUser';
 import { referralService, orderService, offerService } from '../services';
+import { placeOrderOnExchange } from '../services/order.service';
```

Comment:
can use `orderService` var from above import

Reply:
@luckybansal13 VCCode auto adds import when I use placeOrderOnExchange. updated this